Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM nvidia/cuda:12.2.0-devel-ubuntu22.04
```

Comment:
It would be awesome if open-interpreter officially supports both of CPU/GPU images.